libraries/AP_OSD/AP_OSD.h

```diff
@@ -62,6 +62,7 @@ class AP_OSD_Setting
     AP_Int8 xpos;
     AP_Int8 ypos;
 
+    AP_OSD_Setting() {};
```

Comment:
Should the be `CLASS_NO_COPY(AP_OSD_Setting);` to avoid inadvertent copies being made?

Reply:
actually I don't need a default constructor anymore, I can remove it altogether